tests/Unit/Domain/ValueObject/GeographicPositionTest.php

```diff
@@ -12,26 +12,26 @@
 namespace Eluceo\iCal\Unit\Domain\ValueObject;
 
 use Eluceo\iCal\Domain\ValueObject\GeographicPosition;
-use Generator;
 use InvalidArgumentException;
+use PHPUnit\Framework\Attributes\DataProvider;
 use PHPUnit\Framework\TestCase;
 
 class GeographicPositionTest extends TestCase
 {
-    /**
-     * @dataProvider provideInvalidPositions
-     */
+    #[DataProvider('provideInvalidPositions')]
```

Comment:
This is the new recommended way to configure DataProviders.
https://docs.phpunit.de/en/12.1/writing-tests-for-phpunit.html#writing-tests-for-phpunit-data-providers